beetsplug/lyrics.py

```diff
@@ -659,22 +663,69 @@ def commands(self):
             help=u'print lyrics to console',
         )
         cmd.parser.add_option(
+            u'-r', u'--write-rst', dest='writerst',
+            action='store', default='.',
+            help=u'write lyrics to given directory as RST files',
+        )
+        cmd.parser.add_option(
             u'-f', u'--force', dest='force_refetch',
             action='store_true', default=False,
             help=u'always re-download lyrics',
         )
+        cmd.parser.add_option(
+            u'-s', u'--skip', dest='skip_fetched',
+            action='store_true', default=False,
+            help=u'skip already fetched lyrics',
+        )
```

Comment:
For some reason, the option name “skip” seems a little counterintuitive to me. (And “skip already fetched lyrics” is the opposite of what it does, right? It skips lyrics that have *not* been fetched, if I understand correctly.) Maybe another name like “local” or “print-only” or something would be clearer?